src/data_import/mongodb/mongo_wrapper.py

```diff
@@ -0,0 +1,119 @@
+"""
+A simple library for wrapping around mongo collections and access issues.
```

Comment:
Need docstrings and some real-life examples. Here is an example of a suitable docstring with a single atomic example:

https://github.com/r-bioinformatics/edgePy/blob/174eb34224af2ad75a8d1dc0c220b1c3577c087b/edgePy/_DGEList.py#L18-L37